### DISCUSSION TITLE
New install (multiple VMs): Driver download fails, citing: "dbeaver network error network unavailable due to certificate issue"

### DISCUSSION THREAD

**Discussion by gilboa** [dbeaver/dbeaver#23404] [Q&A]
### Description

I'm installing dbeaver on a couple of Windows 10/11 64bit / Pro VMs using the latest CE release (24.0.2) in order to connect to the PostgreSQL DBs.
No issues when upgrading the existing installations.
Across the board (3 VMs) when trying to download the PostgreSQL driver, I get the following error:

_"Network unavailable due to a certificate issue.
Try changing the setting `Use Windows trust store` in Preferences->Connections and restart DBeaver. It might help if you haven't overridden the trust store.
javax.net.ssl.SSLHandshakeException:PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
Network unavailable due to a certificate issue.
Try changing the setting `Use Windows trust store` in Preferences->Connections and restart DBeaver. It might help if you haven't overridden the trust store.
javax.net.ssl.SSLHandshakeException:PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target"_

### DBeaver Version

24.0.2

### Operating System

Windows 10 64bit Pro / 11 64Bit Pro

### Database and driver

PostgreSQL / build-in-driver.

### Steps to reproduce

1. Fresh installation.
2. Create new database connection to a localhost PostgreSQL.
3. Enter database, user and password.
4. DBeaver tries to download the driver.
5. Error

**Comment by gilboa** [comment]
Related to solved issue https://github.com/dbeaver/dbeaver/issues/17591 ?

**Comment by LonwoLonwo** [comment]
Hello @gilboa 

Did you try to follow the advice from the issue message?

> Try changing the setting Use Windows trust store in Preferences->Connections and restart DBeaver. It might help if you haven't overridden the trust store.

**Comment by gilboa** [comment]
> Hello @gilboa
> 
> Did you try to follow the advice from the issue message?
> 
> > Try changing the setting Use Windows trust store in Preferences->Connections and restart DBeaver. It might help if you haven't overridden the trust store.

I did try it... but due to user error on my end, I failed to restart dbeaver after disabling Windows trust store.
Hence it didn't work.
Seems to work just once I disable the Windows trust store and restart dbeaver.
Testing all VMs.

Any ideas why some VMs work w/ trust store enable while others don't?

- Gilboa

**Comment by gilboa** [comment]
--> Many thanks for the prompt help!

**Comment by E1izabeth** [comment]
>Any ideas why some VMs work w/ trust store enable while others don't?

It may depend on the Windows updates

So, the issue is solved now, right?

**Comment by gilboa** [comment]
Yes. Works as advertised. Thanks!

**Comment by dcruz1990** [comment]
Hi, i'm still getting this:
![image](https://github.com/user-attachments/assets/f1f9c75d-6239-4545-82d5-a9604b9122a3)
Using Windows 11 x64
